Integrations/integration-Remedy-On-Demand.yml

```diff
@@ -4,7 +4,8 @@ commonfields:
nufR0NBASclIZs364aCNbyAYnDX6hhpoaEJMYEWDOkGCbQT9wjh0zPbtV/vvv6j2osNVb/oJ0laHjgGbIvETU4PjkVXiogsJ2gBUgRx4OHL/i8hFd4tJ37qY+W2lv+KQ3ek+/e6+v+55XvJn/U37E5Y8FovxySflXHLJRUydOnXYWqBtIScnV667bh7GSDJ4CHpey+7XM6hmk0ooFOrx0Ey8J1Fzrquri9raWq688kpKSkqG9bUcHKF3NCNdYD1IFFwQHzxAxk3GZBZ+6otqf/0L5ZzD0H++EHe4CWAQK0EqqhoUg3rBNh6rFOkMwUXX4t3ygpgv7LwzsXtXZ+no6KCzszMZ5dZ7at/bSvVN5DC88cYbHH30MZx22mmbGcewvmf75aCDDpI5c+ZQU1NNS0tLskY8bJxF9b6OfQOMtFtqL3z44Yd861tHc9xxxw37CzZIXneLekFMjJCo36YQAjWfrktauUztxccot1wNqSCjgqg24xPfLgsKQlqxEIo7/VZYmDgFuf+vmFkD593fEr2n74kbrLi4mMbGRurr60lNTd3kOr77w8Fam8y7fv311zn44EO47bbbdtZQhhTnnTdbrrnm56xdu5bGxsbkNezvuOZNfZ2SkoK1lg8/XM4RRxzGokWLhr3IYZCELhLcmAJo3BmmSOB1j265Vllv7LMPqp4yAxb/BR0PJgeIEpx5jiLWBjXejeIZRRosUg+cdCay8P+Qvb6+w/+YG8skbZxGdp9SNjTUs/feX+Khhx4iHA7z0UcfJbfKgD4nfyasVaKs8scfr+SYY47mscd+S25uzhbG0/PzEzd592XBttC7rZ5bXZatyZbrfW0S4wvatlgbOFC3xNlnny033TSfSKST1avXEI1GexSQ7O8E1cTXIkJDQwPvvPM2xx//PR566FfbfC2GKoOzvdbP313iwWjbMqvUulWqv7oWvfr04P1TCarDxECMh1U/KCBhJBB9BKQWKMyFK24ldORpwqULB2ZMW6Cjo4Pa2mrAEA6n9jknrKqqisrKCqZN21/eeONNvfLKn7Fq1SrS09NJS0uL36wGEZJC6OyM0NnZQX5+PnPnXsUZZ5wmv/71I1vsS1dXlMbGRjo7O5NbSt3ZsGEDGzY0bbGdzs4uPfbYY2lubu5R0ilBcPpqlI6Oji221draQn19Penp6X18EJFIhFgsSmtryxbbATjhhFmyfPmHOn/+zfzrX/8GguCZRABNQvjJ/XLfp7Ozk2g0SmlpKTfffBOnnnqqLFiwYKs+bzgw6Ektnxb75t9Uf3IEfFyOyQebYcBajGpQgcZaDHGRe4rWAE3AMcfAubdgyjZ9csiOoLS0lFNP/TGxWIysrKxeQofm5g1MnjyZRYsWsc8+X5bGxkb9/e//wPPPP0d1dQ0bNmxIvh4gOzubkpJiDjtsOt/61reYOHHrD37Iz8/jpJNOoKurK9mXjRVbDK2t7ey3335bbCctLSwPPPCA7rvvl8nIyOgj9FgsRkZGBoWFhVtsa++99+aMM04jOzsLkscwW1QF348RDoeZOHHi1g6RyZN3E4Bnn31WlyxZwooVK6mpqaWtrTW+1AnaTUtLIzs7m912m8yhhx7CUUcdRVlZ2Wdiut6dQRmQvvaE6mXfRzLAZiS2ugRtVmTakZhrl2y2X/a316n8ci7aZeMntARLgHiUemDFIagYYwWtUEjPwpx5EXLiNcPqj9je3q7Llr3PmjVraWlpSaZglpWVseuuu5KdnT2sxjNY1Nc36AcffEhTU1Myki49PY3c3BwmTpzIpEmTPtPXcVhZdF3zlnLrRdjrfgZjQAsJKsKiWFWQ+PpffTQFNCrwiWL22RcuvR2ZcvCw+2MO17TIoUZBwWcrH39bGTZC9//7LtVzjkQrajBTQA2YriBnXAExgokH32AM0mDRZoUfnQ1nX4dkDt+oJodjexkSFWZ60Gudp221qr84Vpl3fhBos4vBKhA1QdCLBsXdjYINCTZisessZJdgrvs13sX3ixO54/PO4MS6BzEykPhPTbymOuBvPKfLf/cF1XMPQ95+HykBMgTxbdyEJ1JKBTywIkiNDx3AjCOR8+5CRu9ch5vDMVQZnH30REpqkvgZaDHQonEA6O/mq5x7OHz8PnYCaLoJKs9oEAhjCcRNKB5b94lF0nKRK27F3LhEnMgdjo0M0ho97haPW2ZVE1RyCVt0/Ufo1Uep3nw5FIFmCRIlfr5acKpmUJBZIGSQdh/7MZgDvoRe9QBm1/2dwB2OXgyO0OOWOVhjy0axjxD0vdewzQQHKYYFiQqojUe5EWS7GUAUqfKDDNNzL0NPvwSTOdKJ3OHoh8FxxoVT4+txgoB3VQw+akHCIEUE5aSiweksxD3qRg2kCBpRWAdaMga94Q+YOTeJE7nDsWkGxaLbjFwYEcZEuwAbj4JSRMGaeBGKhG8OP6jK6oEai6kGiYDOPBaZfQNSsqsTuMOxBQbFopuCYiSch42SzAvXeB11JDgKSUWxEtSSE09RH1gBaCZ6xa2YeX8UJ3KHY+sYHK97yVShaDzSnEghDYpJCYrYIJU0OFDRQEjQdqBcMfsfAgsXY747cCe0OByfBwavCuwPTg2seSsYUazRIBBGlGR1dWOhTtE6MD+5Cln4ssjU6U7kDsc2Mqii8S+aprrkNcw4AidbYvtMBem0QcbZrruiF96Cd8C3ncAdjk/JoIrH1pYrFxwD77+PpAbx6wB0ASkh+PpRcP5CTEGpE7nDsR0MuoBsY6Xqf9+JfLIcUgViCmKQQ45DDhu4I5gcDofD4XA4HA6Hw+FwOBwOh8PhcDgcDofD4XA4HA6Hw+FwOBwOh8PhcDgcDofD4XA4HA6Hw+FwOBwOh8PhcDgcDofD4XA4HA6Hw+FwOBwOh8PhcDgcDofD4XA4HA6Hw+FwOBwOh8PhcDgcDofD4XA4HA6Hw+FwOBwOh8PhcDgcDofD4XA4HA6Hw+HYFv4/qc9Sn/VcdtAAAAAASUVORK5CYII=
```

Comment:
previous image was default image (puzzle)
I created new image with transparent background that converted to 150X200 pxs